src/content/en/feedback/file-a-bug/index.md

```diff
@@ -0,0 +1,60 @@
+project_path: /web/_project.yaml
+book_path: /web/feedback/_book.yaml
+
+{# wf_updated_on: 2016-10-20 #}
+{# wf_published_on:  #}
+
+# How to file a good bug {: .page-title }
+
+## Decide on the expected behavior
+
+Are there any browsers that implement the feature correctly as a reference? What did you think would happen?
+
+Optionally, you may read about the relevant APIs on [MDN](https://developer.mozilla.org/en-US/) or try to find related specs. This information can help to decide which API is actually broken, where it’s broken, and what the correct behavior should be.
+
+## Write clear reproduction steps
+
+Bugs that get traction tend to be easy to reproduce (i.e. click this link) and easy to understand the expected outcome (eg. screenshots for pass / fail).
```

Comment:
Dru and @foolip (https://github.com/foolip) are working on that document, so I'd redirect questions their way.